docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -1616,7 +1616,7 @@ Once the image is compiled, enable and start JFR via runtime flags: `-XX:+Flight
     -XX:StartFlightRecording="filename=recording.jfr"
 ----
 
-For more details on using JFR, see https://www.graalvm.org/reference-manual/native-image/JFR[here].
+For more details on using JFR, see https://www.graalvm.org/21.3/reference-manual/native-image/JFR[here].
```

Comment:
I wonder if we will need to put the root in an attribute and update it regularly. But I'm unsure the URL are stable.

We are already using a more recent version and moving to 22.2 soon (same comments for all the GraalVM links, there are a few others).

Reply:
Unfortunately URLs are not stable. This link is not valid for 22.2, it has been moved to https://www.graalvm.org/22.2/reference-manual/native-image/debugging-and-diagnostics/JFR/. I think it's best to keep the version embedded in URLs and update it from time to time.

For the time being I will just update the link to point to 22.2 doc.